R/chop.R

```diff
@@ -181,18 +187,26 @@ df_unchop <- function(x, ..., ptype = NULL, keep_empty = FALSE, error_call = cal
       next
     }
 
+    col <- tidyr_new_list(col)
```

Comment:
Necessary for `list_replace_*()` when we have a list-of, since now those aren't allowed (we preserved the ptype earlier on), but this is also part of what fixes the general problem in #1327 that also affects `unchop()`